tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py34, py35, py36
+envlist = py27, py34, py35, py36, py37, py38
```

Comment:
Heh, yeah I forgot to update this in master :) I pushed a separate commit there now, as it shouldn't be concern of this PR.